README.md

```diff
@@ -28,35 +28,37 @@ You can access the [public deployment](https://demo-app.beta.arcana.network) or
 
 * <p>🗝️ &nbsp; Uses social authentication feature of Auth SDK to enable user onboarding via Google OAuth.</p>
 
-* <p>👛 &nbsp;Assigns Arcana wallet address for each authenticated user.</p>
+* <p>👛 &nbsp;Assigns Arcana wallet address for each authenticated user, internally. User doesn't have to necessarily set up and use a wallet explicitly.</p>
 
 * <p>🗄️ &nbsp; Allows dApp users to upload their data in Arcana Store for access control and privacy.</p>
 
 * <p>📂 &nbsp; Enables dApp users full ownership and control of their data residing in Arcana Store. Only data owners can download, share, revoke access, delete or transfer data ownership to another user.</p>
 
-## Public Deployment
+## 🌐 Public Deployment
 
 You can try out Arcana Network Drive here: [https://demo.beta.arcana.network](https://demo.beta.arcana.network)
```

Comment:
We can link to the production demo app: https://demo.arcana.network. It redirects to the beta demo app right now, but this will change in the future.

Reply:
Done.